File: HouseholdPlantsManagement/Models/Plant.cs
namespace HouseholdPlantsManagement.Models
{
    public class Plant
    {
        public int Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Species { get; set; } = string.Empty;
        public DateTime PurchaseDate { get; set; }
        public string ImageUrl { get; set; } = string.Empty;
        public int WateringInterval { get; set; }
        public int FertilizingInterval { get; set; }
        public string Notes { get; set; } = string.Empty;
        public DateTime LastWatered { get; set; }
        public DateTime LastFertilized { get; set; }
    }
}

File: frontend/my-react-app/src/models/Plant.ts
export interface Plant {
  id: number;
  name: string;
  species: string;
  purchaseDate: string;
  imageUrl: string;
  wateringInterval: number;
  fertilizingInterval: number;
  notes: string;
  lastWatered: string;
  lastFertilized: string;
}
